inception-workload/src/main/resources/de/tudarmstadt/ukp/inception/workload/model/db-changelog.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+  <changeSet author="Christoph Scheunemann" id="20200408-01">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <tableExists tableName="workload_manager" />
+      </not>
+    </preConditions>
+    <createTable tableName="workload_manager">
+      <column autoIncrement="true" name="id" type="BIGINT">
+        <constraints primaryKey="true" />
+      </column>
+      <column name="project" type="BIGINT">
+        <constraints nullable="false" />
+      </column>
+        <column name="extensionPointID" type="VARCHAR(50)">
+            <constraints nullable="false" />
+        </column>
+        <column name="traits" type="CLOB(64000)"></column>
```

Comment:
Please compare your changelog file to the others in the project. We use "LONGTEXT" for traits elsewhere.